### DISCUSSION TITLE
Dateset Elegido

### DISCUSSION THREAD

**Discussion by HM9N** [rramosp/ai4eng.v1#240] [00 - General]
Buenos días

Hemos elegido el siguiente dataset para el proyecto: 

https://archive-beta.ics.uci.edu/dataset/180/plants

Quería preguntarles si sí cumple con los requisitos  y qué consejos me darían sobre esa elección.

Gracias

**Comment by rramosp** [comment]
por tamaño sí, pero tiene un formato extraño y no veo que haya una tarea
clara sobre qué es lo que habría que predecir. yo escogería otro

On Mon, Mar 6, 2023 at 8:02 AM Jhon ***@***.***> wrote:

> Buenos días
>
> Hemos elegido el siguiente dataset para el proyecto:
>
> https://archive-beta.ics.uci.edu/dataset/180/plants
>
> Quería preguntarles si sí cumple con los requisitos y qué consejos me
> darían sobre esa elección.
>
> Gracias
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/240>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIL45Z6AV6U5HPONHHDW2XOADANCNFSM6AAAAAAVRC7CKY>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by HM9N** [reply]
Profesor, he elegido otro dataset, ¿podría decirme por favor qué tan viable es para el proyecto? Como tiene tantas instancias, lo ideal sería eliminar muchas y dejar solo 40000 instancias aproximadamente. 

El dataset es el siguiente: https://www.kaggle.com/competitions/microsoft-malware-prediction/

Muchas gracias.

**Comment by rramosp** [comment]
hola jhon ... pon acá una breve descripción del mismo porfa. Qué es lo que
hay que predecir, cuantas filas y columas tienes, y cuantos ficheros

On Mon, Mar 6, 2023 at 11:18 PM Jhon ***@***.***> wrote:

> Profesor, he elegido otro dataset, ¿podría decirme por favor qué tan
> viable es para el proyecto? Como tiene tantas, lo ideal sería eliminar
> muchas y dejar solo 40000 instancias aproximadamente.
>
> El dataset es el siguiente:
> https://www.kaggle.com/competitions/microsoft-malware-prediction/
>
> Muchas gracias.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/240#discussioncomment-5223228>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIKIANHV2SWAJXZ2FMTW22ZHXANCNFSM6AAAAAAVRC7CKY>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by HM9N** [reply]
Hola.

Hay que predecir la probabilidad de que una máquina con Windows se infecte con varias familias de malware, en función de las diferentes propiedades de esa máquina. Tiene 83 columnas las cuales son de los siguientes tipos:

![image](https://user-images.githubusercontent.com/44328943/223441131-ee83f41e-7b89-4ba4-bd9e-ba155152a278.png)

Las filas son millones, por lo cual ya creé un dataset con las primeras 100.000 filas. En las siguientes imágenes muestro un poco el dataset:

![image](https://user-images.githubusercontent.com/44328943/223442590-e7b66ad2-f0ee-4688-8e32-ac55ce14eee8.png)

![image](https://user-images.githubusercontent.com/44328943/223442732-5face5d6-4196-4703-a36a-b141f92645df.png)

![image](https://user-images.githubusercontent.com/44328943/223442833-80c95c28-6b52-4894-a8d0-67aec3701a53.png)

Los datos fueron recopilados por Windows Defender.

Los ficheros que tiene son: 

![image](https://user-images.githubusercontent.com/44328943/223443094-d500cdb8-99e2-4413-bd55-c7f8d9f542a0.png)


Muchas gracias!

**Reply by HM9N** [reply]
También me gustaría recibir su consejo sobre qué tal está el dataset para la gestión de datos.

Gracias.

**Comment by rramosp** [comment]
no entiendo tu pregunta. no das detalles ni adjuntas enlace?

On Wed, Mar 8, 2023 at 8:57 AM Jhon ***@***.***> wrote:

> También me gustaría recibir su consejo sobre qué tal está el dataset para
> la gestión de datos.
>
> Gracias.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/240#discussioncomment-5242028>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZII2NRWQHFWLMS2BAKDW3CF6LANCNFSM6AAAAAAVRC7CKY>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by HM9N** [reply]
En enlace lo adjunté arriba, es este: https://www.kaggle.com/competitions/microsoft-malware-prediction/

Y los detalles también: https://github.com/rramosp/ai4eng.v1/discussions/240#discussioncomment-5228602

**Reply by HM9N** [reply]
![image](https://user-images.githubusercontent.com/44328943/223733699-8c083f3b-e3b8-471b-8f40-419167f17958.png)

**Reply by rramosp** [reply]
ah vale, disculpa ... es que vi el mensaje en correo y no traìa lo anterior ... sí, es un reto interesnate y con una buena variedad de filas y columnas. Si tienes que submuestrear (como ya has hecho) no te preocupes ... lo importante es que muestres el proceso que sigues, no necesariamente que el rendimiento sea el mejor
